src/SIS_transport.F90

```diff
@@ -208,10 +219,12 @@ subroutine ice_cat_transport(CAS, TrReg, dt_slow, nsteps, G, US, IG, CS, uc, vc,
     ! Accumulated diagnostics
     CAS%dt_sum = CAS%dt_sum + dt_adv
     if (allocated(CAS%uh_sum)) then ; do k=1,nCat ; do j=jsc,jec ; do I=isc-1,iec
-      CAS%uh_sum(I,j) = CAS%uh_sum(I,j) + dt_adv * ((uh_pond(I,j,k) + uh_snow(I,j,k)) + uh_ice(I,j,k))
+      CAS%uh_sum(I,j) = CAS%uh_sum(I,j) + dt_adv * ((uh_pond(I,j,k) + uh_snow(I,j,k)) + uh_ice(I,j,k) + \
```

Comment:
Both in this expression for `CAS%uh_sum` and in the subsequent one for `CAS%vh_sum`, we should add parentheses to specify the order of the sums.  As written, the compiler is free to choose the order with which `uh_pond_ice` is included in the sum.  My personal preference would be to combine the `uh_pond` and `uh_pond_ice` first, but what is more important is that we force an order to the sums to help ensure that we get the same answers when compiled for debugging or for optimized runs. 

Also, the use of `\` as the continuation character is not found elsewhere in the SIS2 code.  Please use `&` for consistency.